src/session/src/context.rs

```diff
@@ -289,17 +306,45 @@ impl ConnInfo {
     }
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Default, Clone, Copy)]
+#[repr(u8)]
 pub enum Channel {
-    Mysql,
-    Postgres,
+    #[default]
+    Other = 0,
```

Comment:
I think "Unknown" may be a better name.

Reply:
Sounds good. `Unknown` is mostly for a (de)serialization issue.